packages/libva_intel_driver_hybrid.rb

```diff
@@ -0,0 +1,51 @@
+require 'package'
+
+class Libva_intel_driver_hybrid < Package
+  description 'VA-API implementation for Intel G45 and HD Graphics family'
+  version '2.4.1'
+  compatibility 'i686 x86_64'
+  if ARCH == 'i686', 'x86_64'
```

Comment:
Does this work?  That would be a new one for me.

Reply:
It compiles... I don't have the hardware though.